examples/new_maps.py

```diff
@@ -0,0 +1,289 @@
+maps_comp[1,:,:] = hp.synfast([cl_dust_tt, cl_dust_ee, cl_dust_bb, cl_dust_te, cl_dust_eb, cl_dust_tb], nside, new=True)
+maps_comp[2,:,:] = hp.synfast([cl_sync_tt, cl_sync_ee, cl_sync_bb, cl_sync_te, cl_sync_eb, cl_sync_tb], nside, new=True)
+
+if o.use_pysm:
+    d2 = models("d2", nside) 
+    s1 = models("s1", nside) 
+    c1 = models("c1", nside) 
+
+    map_I_dust, map_Q_dust, map_U_dust = maps_comp[1,:,:]
+    map_I_sync, map_Q_sync, map_U_sync = maps_comp[2,:,:]
+    map_I_cmb,map_Q_cmb,map_U_cmb = maps_comp[0,:,:]
+
+    # Dust
+    d2[0]['A_I'] = map_I_dust
+    d2[0]['A_Q'] = map_Q_dust
+    d2[0]['A_U'] = map_U_dust
+    d2[0]['spectral_index'] = beta_dust
+    d2[0]['temp'] = temp_dust * np.ones(d2[0]['temp'].size) #need array, no const value for temp with PySM
+    # Sync
+    s1[0]['A_I'] = map_I_sync
+    s1[0]['A_Q'] = map_Q_sync
+    s1[0]['A_U'] = map_U_sync
+    s1[0]['spectral_index'] = beta_sync
+    # CMB
+    c1[0]['A_I'] = map_I_cmb
+    c1[0]['model'] = 'pre_computed' #different output maps at different seeds 
+    c1[0]['A_Q'] = map_Q_cmb
+    c1[0]['A_U'] = map_U_cmb
+
+    sky_config = {'dust' : d2, 'synchrotron' : s1, 'cmb' : c1}
+
+    sky = pysm.Sky(sky_config)
+
+    beams=Simons_Observatory_V3_SA_beams() 
+    instrument_config = {
+        'nside' : nside,
+        'frequencies' : nu, #Expected in GHz 
+        'use_smoothing' : o.use_smoothing,
+        'beams' : beams, #Expected in arcmin 
+        'add_noise' : False,
+        'use_bandpass' : False,
+        'channel_names' : ['LF1', 'LF2', 'MF1', 'MF2', 'UHF1', 'UHF2'],
+        'output_units' : 'uK_RJ',
+        'output_directory' : dirname,
+        'output_prefix' : '/test_deltabpass_',
+    }
+    
+    sky = pysm.Sky(sky_config)
+    instrument = pysm.Instrument(instrument_config)
+    maps_signal, _ = instrument.observe(sky, write_outputs=False)
+    maps_signal = maps_signal[:,1:,:]
+    maps_signal = maps_signal/f_cmb_RJ[:,None,None]
+else:
+    sed_comp = np.array([f_cmb, f_dust, f_sync]).T
+    maps_signal = np.sum(maps_comp[None, :,1:,:]*sed_comp[:,:,None,None], axis=1)
+
+# Save map
+nmaps = nfreq * npol
+hp.write_map(dirname+"/maps_sky_signal.fits", maps_signal.reshape([nmaps,npix]),
+             overwrite=True)
+
+# Mask
+nhits=hp.ud_grade(hp.read_map("norm_nHits_SA_35FOV.fits",  verbose=False),nside_out=nside)
+nhits/=np.amax(nhits) 
+fsky_msk=np.mean(nhits) 
+nhits_binary=np.zeros_like(nhits) 
+nhits_binary[nhits>1E-3]=1 
+#inv_sqrtnhits=np.zeros_like(nhits)
+#inv_sqrtnhits[nhits>1E-3]=1./np.sqrt(nhits[nhits>1E-3])
+
+# Noise maps
+nsplits = 4
+maps_noise = np.zeros([nsplits, nfreq, npol, npix])
+for s in range(nsplits):
+    for i in range(nfreq):
+        nell_ee = N_ells_sky[i, 0, i, 0, :]*dl2cl
+        nell_bb = N_ells_sky[i, 1, i, 1, :]*dl2cl
+        nell_00 = nell_ee * 0
+        maps_noise[s, i, :, :] = hp.synfast([nell_00, nell_ee, nell_bb, nell_00, nell_00, nell_00][i]*nsplits, nside, pol=False, new=True) 
```

Comment:
Humm, I think you shouldn't evaluate nell at [i]. Do you agree?

Reply:
Fixed this, thanks!